src/dialog.jsx

```diff
@@ -46,7 +47,7 @@ const TransitionItem = React.createClass({
     this.setState({muiTheme: newMuiTheme});
   },
 
-  componentWillEnter(callback) {
+  componentDidAppear(callback) {
```

Comment:
@chrismcv should this be `componentWillAppear(callback)` instead? https://facebook.github.io/react/docs/animation.html#componentwillappearcallback